FirebaseAdmin/FirebaseAdmin.Tests/Messaging/MulticastMessageTest.cs

```diff
@@ -0,0 +1,1542 @@
+﻿// Copyright 2018, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FirebaseAdmin.Messaging;
+using Google.Apis.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+using Action = FirebaseAdmin.Messaging.Action;
+
+namespace FirebaseAdmin.Tests.Messaging
+{
+    public class MulticastMessageTest
+    {
+        [Fact]
+        public void EmptyMulticastMessage()
+        {
+            var message = new MulticastMessage
+            {
+                Tokens = new[] { "test-token" },
+            };
+            this.AssertJsonEquals(new JObject() { { "tokens", new JArray("test-token") } }, message);
```

Comment:
This object never really gets json serialized directly. We should be unit testing the `GetMessageList()` method here.

Reply:
Totally, per my comment above 🤦‍♂️